### DISCUSSION TITLE
ESP32-ETH01  Ethernet setting

### DISCUSSION THREAD

**Discussion by Forhorse88** [tbnobody/OpenDTU#635] [Q&A] [answered]
Hello,

I'm trying to set up an OpenDTU on a ESP32-ETH01 module but i have some issues with the ethernet connexion. 
The ESP is flashed with a prebuild firmware, then for the device configuration i had uploaded a pin_mapping.json file 

I think there is an error the exemple file wt32-eth01.json 
for the "clock_mode" it's set to "3" but in DeviceProfiles.md it's say that mode 3 =  ETH_CLOCK_GPIO17_OUT
On the WT32-ETH01 the GPIO17 is free to use. The Eth clock pin is GPIO16 (it run successfuly with this seting in ESPeasy)
The right setting for this parameter is mode  2 = ETH_CLOCK_GPIO16_OUT

Can you confirm ?
Thank you

**Comment by tbnobody** [comment, marked as answer]
> The right setting for this parameter is mode 2 = ETH_CLOCK_GPIO16_OUT

Just doublechecked it.. Based on the `pins_arduino.h` file the value should be zero.

![image](https://user-images.githubusercontent.com/1753222/221429489-0a1bc07f-b2a6-4d98-9033-0e2435679be7.png)

I will correct that in the json file

**Comment by Forhorse88** [comment]
In the datasheet https://files.seeedstudio.com/products/102991455/WT32-ETH01_datasheet_V1.1-%20en.pdf
it say that GPIO is for debuging interface (serial TX)
i will try this configuration on my module (but for now it's work in mode 2)

**Comment by Forhorse88** [comment]
eth clock_mode = 0 work very well 
thank you
